core/src/main/java/org/aerogear/mobile/core/metrics/NetworkMetricsPublisher.java

```diff
@@ -37,7 +35,7 @@ public NetworkMetricsPublisher(final Context context, final HttpRequest httpRequ
     }
 
     @Override
-    public void publish(@NonNull String type, @NonNull final Metrics[] metrics,
+    protected void publish(@NonNull String type, @NonNull final Metrics[] metrics,
```

Comment:
Actually this method is used only internally so not sure why needs to be protected.

Reply:
good catch, I thought I was getting compile errors by turning this package-private but changing the scope of the parent method should be okay :+1: